tests/e2e/hci_mesh_rwx_singlevc_topology.go

```diff
@@ -80,17 +88,30 @@ var _ = ginkgo.Describe("[hci-mesh-rwx-topology] Hci-Mesh-Topology-SingleVc", fu
 		labelsMap = make(map[string]string)
 		accessmode = v1.ReadWriteMany
 		bindingModeWffc = storagev1.VolumeBindingWaitForFirstConsumer
-		//bindingModeImm = storagev1.VolumeBindingImmediate
+		bindingModeImm = storagev1.VolumeBindingImmediate
 
 		//read topology map
-		topologyLength, leafNode, _, _, leafNodeTag2 = 5, 4, 0, 1, 2
+		/*
+			here 5 is the actual 5-level topology length (ex - region, zone, building, level, rack)
+			4 represents the 4th level topology length (ex - region, zone, building, level)
+			here rack further has 3 rack levels -> rack1, rack2 and rack3
+			0 represents rack1
+			1 represents rack2
+			2 represents rack3
+		*/
+		topologyLength, leafNode, _, leafNodeTag1, leafNodeTag2 = 5, 4, 0, 1, 2
 		topologyMap := GetAndExpectStringEnvVar(topologyMap)
```

Comment:
defined it as envTopologyMap, as it is a env variable

Reply:
fixed